src/main/java/com/solvd/project/carina/demo/gui/pages/admin/job/JobTitlePage.java

```diff
@@ -0,0 +1,53 @@
+package com.solvd.project.carina.demo.gui.pages.admin.job;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.PageOpeningStrategy;
+import com.solvd.project.carina.demo.gui.components.JobDeleteOptionPanel;
+import com.solvd.project.carina.demo.gui_components.abstractclass.AbstractOrangeHRMPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class JobTitlePage extends AbstractOrangeHRMPage {
+
+    @FindBy(css = ".oxd-dialog-sheet--shadow")
+    private JobDeleteOptionPanel jobDeleteOptionPanel;
+
+    @FindBy(xpath = "//*[@class = 'oxd-text oxd-text--h6 orangehrm-main-title'][text() = 'Job Titles']")
+    private ExtendedWebElement formTitle;
+
+    @FindBy(xpath = "//*[text() = ' Add ']")
+    private ExtendedWebElement addBtn;
+
+    @FindBy(xpath = "//*[@class = 'oxd-table-cell oxd-padding-cell']//*[text() = '%s']")
+    private ExtendedWebElement jobTitles;
```

Comment:
```suggestion
    private ExtendedWebElement jobTitle;
```
fix it if this locator searches for one element